From a0bb1d1bac5811d93a7a53dbfc1dac73ed33c706 Mon Sep 17 00:00:00 2001
From: Xianzhong <b07117@freescale.com>
Date: Thu, 29 Aug 2013 23:52:00 +0800
Subject: [PATCH 07/14] ENGR00277045-2 align gpu baseaddr with ram base address

2G above address will be converted to gpu virtual address in i.MX6,
gpu baseaddr can be used to reduce contiguous memory address conversion,
the benifit for gpu applications and external memory consumers(VPU/IPU) have,
 - gpu mmu performance penalty is reduce in most cases,
 - more system memory can be exported for external use

Signed-off-by: Xianzhong <b07117@freescale.com>
---
 arch/arm/plat-mxc/devices/platform-viv_gpu.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/plat-mxc/devices/platform-viv_gpu.c b/arch/arm/plat-mxc/devices/platform-viv_gpu.c
index 52c9b08..677a747 100644
--- a/arch/arm/plat-mxc/devices/platform-viv_gpu.c
+++ b/arch/arm/plat-mxc/devices/platform-viv_gpu.c
@@ -34,7 +34,7 @@ const struct imx_viv_gpu_data imx6_gpu_data __initconst = {
 };
 #else
 const struct imx_viv_gpu_data imx6_gpu_data __initconst = {
-	.phys_baseaddr = 0,
+	.phys_baseaddr = MMDC0_ARB_BASE_ADDR,
 	.iobase_3d = GPU_3D_ARB_BASE_ADDR,
 	.irq_3d = MXC_INT_GPU3D_IRQ,
 	.iobase_2d = GPU_2D_ARB_BASE_ADDR,
-- 
1.9.1

